aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -131,12 +133,12 @@ const StepOne = () => {
     WizardContext,
   );
   const size = useContext(ResponsiveContext);
-  console.log(error.isValid);
+
   return (
     <>
       <Box margin={{ bottom: 'medium' }}>
```

Comment:
Can this margin be applied to the Grid itself? I'm trying to see if we can minimize the number of Boxes

Reply:
Actually, it looks like this margin-bottom is desired across steps. It looks like the desire is to have a medium space between the StepContent and StepFooter. If so, can we eliminate each individual instance and handle that by placing a Box with gap around
```
<Box gap="medium">
  <StepContent />
  <StepFooter />
</Box>
```

Reply:
Thanks let me have a look into it.

Reply:
I think we want to keep with bottom margin, otherwise the scroll of the content doesn't reach all the way to the footer. There is a medium gap of white which makes the content look like it's broken up strangely.